### DISCUSSION TITLE
Request for Assistance with Student Benefits Activation

### DISCUSSION THREAD

**Discussion by Romnoms** [community/community#134979] [GitHub Education]
### Hi everyone,

Hello GitHub Education Team,

I hope this message finds you well.

I am writing to seek your assistance regarding my recent application for student benefits. My application has been approved, but I have not yet received the student benefits, even though three days have passed since the approval.

Could you please look into this matter and help resolve any issues that may be causing the delay?

Thank you for your attention to this matter.

**Comment by Irissuu** [comment]
check your email for any updates, go to the [gitHub education page](https://education.github.com/pack) and try logging in there to see if you can access the benefits, if you still haven't received any instructions, reach out to gitHub support. You can contact them through their [support page](https://support.github.com/)

**Comment by prabhjotschugh** [comment]
Hi @Romnoms ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks;

When the benefits become available, the status label will turn purple and change to:

`Congrats! Your request for @Romnoms was approved and your benefits are now available.`

For further reference you can look at these discussions: [#132790](https://github.com/orgs/community/discussions/132790) , [#131665](https://github.com/orgs/community/discussions/131665), [#132470](https://github.com/orgs/community/discussions/132470) , [#133531](https://github.com/orgs/community/discussions/133531), [#134279](https://github.com/orgs/community/discussions/134279)

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

[How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
[How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
